src/js/mixins/TabsMixin.js

```diff
@@ -93,16 +94,27 @@ const TabsMixin = {
    * @return {Component} React component to render
    */
   tabs_getRoutedItem(customClasses, tab) {
+    let notificationCount = NotificationStore.getNotificationCount(tab);
```

Comment:
hmm, good point. most of our tab names are named exactly the same as the route name, but there are a few tabs that aren't routed yet. what do you think we should do?